core/src/main/java/kafka/server/TierStateMachine.java

```diff
@@ -17,15 +17,66 @@
 
 package kafka.server;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.Optional;
```

Comment:
`:core:checkstyleMain`  is failing because this import is unused

```

> Task :core:checkstyleMain FAILED
[ant:checkstyle] [ERROR] 
(...)/kafka/core/src/main/java/kafka/server/TierStateMachine.java:33:8: Unused import - java.util.Optional. [UnusedImports]
```